dotnetv3/cross-service/AuroraItemTracker/Readme.md

```diff
@@ -0,0 +1,180 @@
+
+## Prerequisites
+
+To run the code in this example, you need the following:
+
++ An AWS account.
++ .NET 6
++ To set up your development environment,
+see [Setting up your AWS SDK for .NET environment](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-setup.html). 
+
+## Create the resources
+
+Using the AWS AWS Cloud Development Kit (AWS CDK), you can set up the resources required for this example. For more information, see [CDK instructions](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/cdk/aurora_serverless_app/README.md).
+
+### Create the database table
+
+After you have created the Aurora DB cluster and database, create a table to contain work items. You can do this by using either the AWS Command Line Interface (AWS CLI) or the AWS Management Console.
+AWS CLI
+
+Use the AWS CLI to create the work_items table by running the following command at a command prompt. Before you run, replace the following values with the output from the CloudFormation setup script:
+
+- **CLUSTER_ARN** — Replace with the ARN of the Aurora DB cluster, such as `arn:aws:rds:us-west-2:123456789012:cluster:doc-example-aurora-app-docexampleauroraappcluster-15xfvaEXAMPLE`.
+- **SECRET_ARN** — Replace with the ARN of the secret that contains your database credentials, such as `arn:aws:secretsmanager:us-west-2:123456789012:secret:docexampleauroraappsecret8B-xI1R8EXAMPLE-hfDaaj`.
+- **DATABASE** — Replace with the name of the database, such as `auroraappdb`.
+
+Tip: The caret ^ is the line continuation character for a Windows command prompt. If you run this command on another platform, replace the caret with the line continuation character for that platform.
+
+```
+aws rds-data execute-statement ^
+    --resource-arn "CLUSTER_ARN" ^
+    --database "DATABASE" ^
+    --secret-arn "SECRET_ARN" ^
+    --sql "CREATE TABLE items (id VARCHAR(45), description VARCHAR(400), guide VARCHAR(45), status VARCHAR(400), name VARCHAR(45), archived BOOLEAN);"
+```
+
+### Verified email address
+
+To email reports from the app, you must register at least one email address with Amazon SES. This verified email is specified as the sender for emailed reports.
+
+1. In a browser, navigate to the Amazon SES console.
+1. If necessary, select your AWS Region.
+1. Select Verified identities.
+1. Select Create identity.
+1. Select Email address.
+1. Enter an email address you own.
+1. Select Create identity.
+1. You will receive an email from Amazon Web Services that contains instructions on how to verify the email with Amazon SES. Follow the instructions in the email to complete verification.
+
+Tip: For this example, you can use the same email account for both the sender and the recipient.
+
+## Build the code
+
+This application has two parts: a user interface that uses React.js and a
+RESTful API created with C# and .NET 6. The React user interface is a single-page
```

Comment:
Standardize usage on either React.js or React (I think React is more standard based on web searching).

Reply:
Updated to use React.